### DISCUSSION TITLE
Grid does not reach the edges of the image

### DISCUSSION THREAD

**Discussion by ivanvinicius** [ValentinH/react-easy-crop#479] [Q&A]
I implemented Cropper inside a Radix Dialog and the problem I'm having is:

![Screenshot from 2023-08-03 20-43-43](https://github.com/ValentinH/react-easy-crop/assets/52886299/483d92bc-fef9-467c-8591-0402eec154f7)

![Screenshot from 2023-08-03 20-25-43](https://github.com/ValentinH/react-easy-crop/assets/52886299/1518d2ee-0b1f-45bd-b393-e6d828ceaa2c)

My grid does not reach the edges of the image. I believe it's a CSS issue that I haven't identified. And why do I say that?

Basically when I change width property (on cropp container or grid) while the Cropper is open, the grid works correctly again

Here's my gist containing the code of component and styles, using stitchesjs

[GIST](https://gist.github.com/ivanvinicius/7d2e2cf60abb7e37e71823fa53592b69)

**Comment by ValentinH** [comment]
It's probably a measurement issue while the dialog is being opened.
If you resize the window while the issue is present, does it fix it?

If yes, it's a common issue with opening animations, you can search for issues with the dialog or modal keyword.

**Reply by ivanvinicius** [reply]
My layout is not responsive, so I defined fixed width and height for the dialog, and its content. 

I tried to test resizing the window, but nothing happens.

I will look for something about your mentioned tip above.
